### PR TITLE
ARMADA-799 Update charm for next-gen jobbergate

### PR DESCRIPTION
**Pull request by fschuch**
#### What
Update charm for next-gen jobbergate.

#### Why
Need to have a branch targeting the next-gen jobbergate-cli that includes these updates.
The main branch should continue to support the legacy jobbergate-cli.

`Task`: https://app.clickup.com/t/18022949/ARMADA-799

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

**Review by dusktreader — APPROVED** [review]
This all looks great to me.

**Review by dusktreader — APPROVED** [review]

**Comment by matheushent** [comment]
It's been very weird for me to have it named as `new-jobbergate`. I think just jobbergate-v3 would be better

**Comment by matheushent** [comment]
> It's been very weird for me to have it named as `new-jobbergate`. I think just jobbergate-v3 would be better

Specially because, as far as I tested here, I still need to run `jobbergate` to invoke the commands

**Comment by fschuch** [comment]
> > It's been very weird for me to have it named as `new-jobbergate`. I think just jobbergate-v3 would be better
> 
> Specially because, as far as I tested here, I still need to run `jobbergate` to invoke the commands

Do you mean we need a way to rename the executable command as well? Any suggestions on that?

**Review by dusktreader — APPROVED** [review]
One question for you, but looks good besides.

File: src/jobbergate_cli_ops.py [hunk 1]
```diff
@@ -30,10 +29,6 @@ def __init__(self, charm):
     def install(self):
         """Install package from private pypi."""
 
-        # Create log dir
-        if not self._LOG_DIR.exists():
```
[the remark this answers is not in the document]
Comment (fschuch) [reply src/jobbergate_cli_ops.py:40]: The next-gen overwrites anything there:

https://github.com/omnivector-solutions/jobbergate/blob/a146bc91407ef786e63d84875d653d868ec8f125/jobbergate-cli/jobbergate_cli/config.py#L74-L76

**Pull request merged**